types/forge-viewer/index.d.ts

```diff
@@ -2910,16 +2911,22 @@ declare namespace Autodesk {
             scale?: number | undefined,
             url?: string | undefined
           }): void;
-        registerSurfaceShadingColors(sensorType: string, colors: number[], alpha?: number): void;
+        registerSurfaceShadingColors(sensorType: string, colors: number[], options?: {
+            alpha?: number;
+            model?: Viewing.Model;
+        }): void;
         removeAllViewables(): void;
-        removeSurfaceShading(): void;
+        removeSurfaceShading(model?: Viewing.Model): void;
         renderSurfaceShading(nodeIds: string|string[],
           sensorType: string,
           valueCallback: (device: DataVisualization.Core.SurfaceShadingPoint, sensorType: string) => number,
-          heatmapConfig?: {
-            confidence?: number | undefined;
-            powerParameter?: number | undefined;
-            alpha?: number | undefined;
+          options?: {
```

Comment:
Has the method signature really changed from `renderSurfaceShading(..., ..., ..., { confidence: 0.5, ... })` to `renderSurfaceShading(..., ..., ..., { heatmapConfig: { confidence: 0.5, ... } })`? If so, I'm afraid that is a breaking change.